core/keyserver/keyserver_test.go

```diff
@@ -1,267 +0,0 @@
-// Copyright 2016 Google Inc. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package keyserver
-
-import (
-	"bytes"
-	"database/sql"
-	"encoding/gob"
-	"errors"
-	"fmt"
-	"testing"
-
-	"github.com/google/keytransparency/core/appender"
-	"github.com/google/keytransparency/core/authentication"
-	"github.com/google/keytransparency/core/fake"
-	"github.com/google/keytransparency/core/mapserver"
-	"github.com/google/keytransparency/core/transaction"
-	"github.com/google/trillian"
-
-	"github.com/golang/protobuf/proto"
-	"golang.org/x/net/context"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/codes"
-
-	authzpb "github.com/google/keytransparency/core/proto/authorization"
-	tpb "github.com/google/keytransparency/core/proto/keytransparency_v1_types"
-)
-
-const (
-	logID = int64(0)
-	mapID = int64(0)
-)
-
-func TestListEntryHistory(t *testing.T) {
-	profileCount := 25
-	ctx := context.Background()
-	for i, tc := range []struct {
-		start       int64
-		page        int32
-		wantNext    int64
-		wantHistory []int
-		err         codes.Code
-	}{
-		{1, 1, 2, []int{1}, codes.OK},                                                            // one entry per page.
-		{0, 1, 2, []int{1}, codes.OK},                                                            // start epoch is not set (will default to 1).
-		{1, 10, 11, []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}, codes.OK},                              // 10 entries per page.
-		{4, 10, 14, []int{4, 5, 6, 7, 8, 9, 10, 11, 12, 13}, codes.OK},                           // start epoch is not 1.
-		{1, 0, 17, []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}, codes.OK},       // zero page size.
-		{20, 10, 0, []int{20, 21, 22, 23, 24}, codes.OK},                                         // end of list.
-		{24, 10, 0, []int{24}, codes.OK},                                                         // requesting the very last entry.
-		{1, 1000000, 17, []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}, codes.OK}, // DOS prevention.
-		{40, 10, 0, []int{}, codes.InvalidArgument},                                              // start epoch is beyond current epoch.
-		{-1, 1, 1, []int{0}, codes.InvalidArgument},                                              // start epoch is less than 0.
```

Comment:
Shouldn't this test be moved to `integration` instead of deleting it? If yes, I'm happy if we merge this pull an open an issue for that (I can give it a shot).